use_cases/kg_chat/backend/src/main.py

```diff
@@ -38,28 +43,27 @@ class Payload(BaseModel):
 
 
 neo4j_connection = Neo4jDatabase(
-    host=os.environ.get("NEO4J_URL", "bolt://neo4j:7687"),
-    user=os.environ.get("NEO4J_USER", "neo4j"),
-    password=os.environ.get(
-        "NEO4J_PASS", "pleaseletmein")
+    host="bolt://neo4j:7687", user="neo4j", password="pleaseletmein"
```

Comment:
any specific reason for removing the env variable usage here?

Reply:
Removed them during testing. Same reason I pushed the API key. Will fix